Src/Logic/APIDetailsStorage/APIDetailsStorage.ts

```diff
@@ -0,0 +1,39 @@
+import { DetailsAPI } from "../../../Models/DetailsAPI.model";
+import { saveInLocalStorage, readFromLocalStorage } from "../../../Src/Logic/LocalStorage/LocalStorage"
+
+const NO_USER_ERROR = `User doesn't exist!`;
+const NO_API_DETAILS_ERROR = `User doesn't contain API details.`;
+const SAVED_CORRECTLY_CONFIRM = 'API DETAILS saved correctly!';
```

Comment:
I suggest to move these lines to new consts.ts file.
And import const.ts to this file and test.ts file.

Reply:
That's great suggestion but I am not sure in terms of project architecture where I should place that file.
New folder 'Constants' in main directory and over there file like LocalStorage.ts?
Or just 'const.ts' in ./Src/ ?

Reply:
In JS project contants files were added to file next to component file.
I think this question should be adressed to Tech Lead @JustynaSmoliga

Reply:
I think, that we should chose option "New folder 'Constants' in main directory and over there file like const.ts".